golang/cli_test/README.md

```diff
@@ -0,0 +1,56 @@
+# Relay CLI Integration tests
+
+The relay cli integration tests live in this folder. You can run the full suite by running:
+
+```bash
+go test -mod=readonly -p 4 `go list ./cli_test/...`
+```
+
+ To run a single test run:
+ ```bash
+ go test -mod=readonly -p 4 `go list ./cli_test/...` -testify.m TestName
+ ```
+
+> NOTE: While the full suite runs in parallel, some of the tests can take up to a minute to complete
+
+### Test Structure
+
+This integration suite [uses a thin wrapper](https://godoc.org/github.com/cosmos/cosmos-sdk/tests) over the [`os/exec`](https://golang.org/pkg/os/exec/) package. This allows the integration test to run against built binaries (both `relayd` and `relaycli` are used) while being written in golang. This allows tests to take advantage of the various golang code we have for operations like marshal/unmarshal, crypto, etc...
+
+> NOTE: The tests will use whatever `relayd` or `relaycli` binaries are available in your `$PATH`. You can check which binary will be run by the suite by running `which relayd` or `which relaycli`. If you have your `$GOPATH` properly setup they should be in `$GOPATH/bin/relay*`. This will ensure that your test uses the latest binary you have built
```

Comment:
This note is not quite right. 

The tests try to use the $BUILDDIR env variable, and error if binaries aren't found in that location. If it's blank, they check to see if binaries are in $GOPATH/bin. If so, they use those. If not, they attempt to use `../build/` and error if binaries aren't found in that location.

Reply:
I almost wanna leave this as is, and take out the `$BUILDDIR` `../build/`  attempts and just use the binaries as built with `make install`. It seems excessive but maybe I'm missing a use case here. Or I can update the README. lmk what you think

Reply:
Hmmm, as-is is not quite right. Because we don't actually search the `$PATH` at the moment. We search `$GOPATH/bin`

Two solutions. #1 is easier.
1. Update text to say `$GOPATH/bin` and delete the code for `$BUILDDIR` and `../build`
2. Keep text as is, and check `$PATH` by running `which relayd` in the go code before checking `$GOPATH`. Delete `$BUILDDIR` and `../build`

Reply:
#1 done ✅